src/pybind/mgr/dashboard/services/access_control.py

```diff
@@ -632,6 +635,25 @@ def ac_user_set_password(_, username, password):
         return -errno.ENOENT, '', str(ex)
 
 
+@CLIWriteCommand('dashboard ac-user-set-password-hash',
+                 'name=username,type=CephString '
+                 'name=hashed_password,type=CephString',
+                 'Set user password bcrypt hash')
+def ac_user_set_password_hash(_, username, hashed_password):
+    try:
+        # make sure the hashed_password is actually a bcrypt hash
+        bcrypt.checkpw(b"", hashed_password.encode('utf-8'))
```

Comment:
We do not test this righ at the moment, but if you use single quotes everywhere, why not here, too :-)

```suggestion
        bcrypt.checkpw(b'', hashed_password.encode('utf-8'))
```

Reply:
Fastest resolve in the west 🤠